src/defaultPhrases.js

```diff
@@ -1,9 +1,36 @@
 const closeDatePicker = 'Close';
-const focusStartDate = 'Focus on start date';
+const focusStartDate = 'Interact with the calendar and add the check-in date for your trip.';
 const clearDate = 'Clear Date';
 const clearDates = 'Clear Dates';
-const jumpToPrevMonth = 'Jump to previous month';
-const jumpToNextMonth = 'Jump to next month';
+const jumpToPrevMonth = 'Move backward to switch to the previous month';
+const jumpToNextMonth = 'Move forward to switch to the next month';
+const keyboardShortcuts = 'Keyboard Shortcuts';
+const showKeyboardShortcutsPanel = 'Open the keyboard shortcuts panel';
+const hideKeyboardShortcutsPanel = 'Close the shortcuts panel';
+const openThisPanel = 'Open this panel';
+const enterKey = 'Enter key';
+const leftArrowRightArrow = 'Right and left arrow keys';
+const upArrowDownArrow = 'up and down arrow keys';
+const pageUpPageDown = 'page up and page down keys';
+const homeEnd = 'Home and end keys';
+const escape = 'Escape key';
+const questionMark = 'Question mark';
+const selectFocusedDate = 'Select the date in focus';
+const moveFocusByOneDay = 'Move backward (left) and forward (right) by one day';
+const moveFocusByOneWeek = 'Move backward (up) and forward (down) by one week';
+const moveFocusByOneMonth = 'Switch months';
+const moveFocustoStartAndEndOfWeek = 'Go to the first or last day of a week';
+const returnFocusToInput = 'Return to the date input field';
+const keyboardNavigationInstructions = `Press the down arrow key to interact with the calendar and
+  select a date. Press the question mark key to get the keyboard shortcuts for changing dates.`;
+
+// eslint-disable-next-line camelcase
+const chooseAvailableStartDate = ({ checkin_date }) => `Choose ${checkin_date} as your check-in date. It's available.`;
+
+// eslint-disable-next-line camelcase
+const chooseAvailableEndDate = ({ checkout_date }) => `Choose ${checkout_date} as your check-out date. It's available.`;
```

Comment:
So when you look at `getPhrase`, you'll see that each phrase accepts a function, node, or string and behaves accordingly. My plan is to add code to our wrappers that automatically pulls in our translations appropriately and create a phrase bundle that can easily be spread onto any page.

We might want to do this for the more base-level components (ie DayPickerRangeController) we're using as well.